CMakeLists.txt

```diff
@@ -12,7 +12,8 @@ if (CMAKE_SOURCE_DIR STREQUAL CMAKE_CURRENT_SOURCE_DIR)
   get_target_property(CLANG_BIN clang LOCATION)
   execute_process(COMMAND ${CLANG_BIN} -print-resource-dir
                   OUTPUT_VARIABLE CLANG_INSTALL_RESOURCE_DIR)
-  set(CLANG_RESOURCE_DIR ${CLANG_INSTALL_RESOURCE_DIR}/include)
+  string(REGEX REPLACE "\n" "" CLANG_RESOURCE_DIR
```

Comment:
There's an `OUTPUT_STRIP_TRAILING_WHITESPACE` option to `execute_process`, would that fix this? Also, please squash this into the `Use the imported clang binary [...]` commit.